frontend/src/middleware/middleware.ts

```diff
@@ -33,9 +33,6 @@ const middleware: Middleware = async ({
 
   const navigationStore = useNavigationStore($pinia)
 
-  if ("embedded" in query) {
-    navigationStore.setIsEmbedded(query.embedded === "true")
-  }
   if (process.client) {
```

Comment:
```suggestion
  if (process.client) {
```
Could you also remove the `sendWindowMessage`?